### DISCUSSION TITLE
Terminal is it fast as Alacritty?

### DISCUSSION THREAD

**Discussion by Grabber** [zed-industries/zed#6502] [Feature Requests]
How fast is it? Are you planning to release a separated terminal app?

**Comment by LeeeSe** [comment]
It's Alacritty

**Comment by dtinth** [comment]
https://zed.dev/features#terminal

![image](https://user-images.githubusercontent.com/193136/226246592-5fb4f608-c478-4a76-8a3f-ed04bbad13cf.png)
